pkg/webui/views/console/device-overview/index.js

```diff
@@ -0,0 +1,152 @@
+  rootKeyId: 'Root Key Id',
+  sessionInfo: 'Session Info',
+  fwdNtwkKey: 'FNwkSIntKey',
+  sNtwkSIKey: 'SNwkSIntKey',
+  ntwkSEncKey: 'NwkSEncKey',
+  appSKey: 'AppSKey',
+})
+
+@connect(function ({ device }, props) {
+  return {
+    device: device.device,
+  }
+})
+@bind
+class DeviceOverview extends React.Component {
+
+  get deviceInfo () {
+    const {
+      ids,
+      description,
+      version_ids,
+      root_keys,
+      session,
+      created_at,
+    } = this.props.device
+
+    const {
+      f_nwk_s_int_key,
+      s_nwk_s_int_key,
+      nwk_s_enc_key,
+      app_s_key,
+    } = session.keys
+
+    const sheetData = [
+      {
+        header: sharedMessages.generalInformation,
+        items: [
+          { key: sharedMessages.devID, value: ids.device_id, type: 'code', sensitive: false },
+          { key: sharedMessages.description, value: description || sharedMessages.noDesc },
+          { key: sharedMessages.createdAt, value: created_at.toLocaleDateString() },
+        ],
+      },
+      {
+        header: m.hardware,
+        items: [
+          { key: m.brand, value: version_ids.brand_id },
+          { key: m.model, value: version_ids.model_id },
+          { key: m.hardwareVersion, value: version_ids.hardware_version },
+          { key: m.firmwareVersion, value: version_ids.firmware_version },
+        ],
+      },
+      {
+        header: m.activationInfo,
+        items: [
+          { key: sharedMessages.devEUI, value: ids.dev_eui, type: 'byte', sensitive: false },
+          { key: sharedMessages.joinEUI, value: ids.join_eui, type: 'byte', sensitive: false },
+          {
+            key: m.rootKeyId,
+            value: root_keys.root_key_id,
+            type: 'code',
+            sensitive: false,
+            subItems: [
+              { key: sharedMessages.appKey, value: root_keys.app_key.key, type: 'code', sensitive: true },
+              { key: sharedMessages.networkKey, value: root_keys.nwk_key.key, type: 'code', sensitive: true },
+            ],
+          },
+        ],
+      },
+      {
+        header: m.sessionInfo,
+        items: [
+          { key: sharedMessages.devAddr, value: ids.dev_addr, type: 'byte', sensitive: false },
+          { key: m.fwdNtwkKey, value: f_nwk_s_int_key.key, type: 'code', sensitive: true },
+          { key: m.sNtwkSIKey, value: s_nwk_s_int_key.key, type: 'code', sensitive: true },
+          { key: m.ntwkSEncKey, value: nwk_s_enc_key.key, type: 'code', sensitive: true },
+          { key: m.appSKey, value: app_s_key.key, type: 'code', sensitive: true },
+        ],
+      },
+    ]
+
+    return (
+      <div className={style.overviewInfo}>
+        <div>
+          <DataSheet data={sheetData} />
+        </div>
+      </div>
+    )
+  }
+
+  render () {
+    const { device } = this.props
+    const { device_id } = device.ids
+    return (
+      <Container>
+        <IntlHelmet>
+          <title>{device.name || device_id}</title>
```

Comment:
With the ID being referenced clearly in the "General Info" bit, I think it's ok to use this approach here. Otherwise, there is almost no need to have a `name` field at all.